flink-runtime/src/main/java/org/apache/flink/runtime/state/IncrementalKeyedStateHandle.java

```diff
@@ -38,5 +38,5 @@
      * created.
      */
     @Nonnull
-    Set<StateHandleID> getSharedStateHandleIDs();
+    Map<StateHandleID, StreamStateHandle> getSharedStateHandleIDs();
```

Comment:
I guess `ID` has to be removed from the method name now..

Reply:
Agree.